### DISCUSSION TITLE
Adding 16-channel Analog Mux Newbie Question

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#271] [Q&A]
Hi!

I am very happily working with Opendeck running on a Raspberry Pi Pico.

After a couple of failed attempts at using a Teensy with Python for midi, Opendeck is such a breeze.

I loaded the software onto the Pico, opened up the configurator, and everything works for me despite a complete lack of electronics knowledge.

I do have one question...

I need more potentiometers for my project, and the Pico is limited to three potentiometers according to the very helpful OpenDeck wiki.

I bought this 32-channel analog "mux";

https://www.roxxxtar.com/badassmidi-anamux

which is based on two 74HC4067 Analog Multiplexers.

My question is how to wire it to the Pico, and do I have to load more software to the Pico besides OpenDeck to have it work and the configurator to see it?

Thanks!

**Comment by paradajz** [comment]
Check the wiki for info on how to do that:
https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant#analog

You'll also have to build the custom firmware and load it up on the Pico.

**Reply by NRKorsakov** [reply]
Thank you!

**Comment by Bosswhite12** [comment]
This contributor kindly added some Mux Uf.2  files to a thread awhile ago.

https://github.com/shanteacontrols/OpenDeck/discussions/177#discussioncomment-5063095
